pkg/ami/api.go

```diff
@@ -86,6 +86,10 @@ func FindImage(api ec2iface.EC2API, namePattern string) (string, error) {
 		},
 	}
 
+	if imageFamily == "ImageFamilyAmazonLinux2" {
```

Comment:
I think we should match family to owner elsewhere, perhaps we can look it up from another map. But from the standpoint if this function, it's unaware of families of what now, so we just want to pass the owner to it.

Reply:
Both owner IDs are documented here already:

https://github.com/weaveworks/eksctl/blob/93cc5eb09b06a7b60ae6cba123fafd82c001be71/pkg/ami/api.go#L17-L21

Please use the Ubuntu one also, and after you have a lookup map, remove those comments.